.dockleconfig

```diff
@@ -3,4 +3,4 @@
 # DOCKLE_ACCEPT_FILES="file1,path/to/file2,file3/path,etc"
 # https://github.com/goodwithtech/dockle#accept-suspicious-environment-variables--files--file-extensions
 # The apiflask/settings file is a stub file that apiflask creates, and has no sensitive data in. We are ignoring it since it is unused
-DOCKLE_ACCEPT_FILES=api/.venv/lib/python3.11/site-packages/apiflask/settings.py
+DOCKLE_ACCEPT_FILES=api/.venv/lib/python3.12/site-packages/apiflask/settings.py
```

Comment:
Can you explain why this is needed?

Reply:
we're updating from python 3.11 to 3.12. settings.py files normally trigger dockle that this is a sensitive file checked into the codebase. We're telling it to ignore this file, since we're no longer using python 3.11 and instead using python 3.12, we need to update the hardcoded path for the python settings file for dockle to ignore.